RELEASES.md

```diff
@@ -1,3 +1,123 @@
+Version 1.81.0 (2024-09-05)
+==========================
+
+<a id="1.81.0-Language"></a>
+
+Language
+--------
+
+- [Abort on uncaught panics in `extern "C"` functions.](https://github.com/rust-lang/rust/pull/116088/)
+- [Fix ambiguous cases of multiple `&` in elided self lifetimes.](https://github.com/rust-lang/rust/pull/117967/)
+- [Stabilize `#[expect]` for lints (RFC 2383),](https://github.com/rust-lang/rust/pull/120924/) like `#[allow]` with a warning if the lint is _not_ fulfilled.
```

Comment:
```suggestion
- [Stabilize `#[expect]` for lints (RFC 2383)](https://github.com/rust-lang/rust/pull/120924/), like `#[allow]` with a warning if the lint is _not_ emitted.
```

Reply:
I find that confusing too, because this is *preventing* the lint from being emitted. How about:
```suggestion
- [Stabilize `#[expect]` for lints (RFC 2383),](https://github.com/rust-lang/rust/pull/120924/) like `#[allow]` with a warning if the lint would not otherwise be emitted.
```